File: day 1/day1.jl
fileLoc = "C:/Users/Mitja/Documents/AOC24/AOC24/day 1/day1_input.txt"

f = open(fileLoc, "r")
 
# to count total lines in the file
line_count = 0             
list1 = []
list2 = []

for lines in readlines(f)
    val1, val2 = split(lines, " ")
    push!(list1,val1)
    push!(list2,val2)
    # increment line_count
    global line_count += 1 
    # print the line
    println(val1)
end
 
# total lines in file
println("line count is $line_count")
 
close(f)

File: day 1/day1.py
list1 = []
list2 = []
fileLoc = r'C:/Users/Mitja/Documents/AOC24/AOC24/day 1/day1_input.txt'
with open(fileLoc,'r') as f:   
    for line in f:
        values = line.split()
        list1.append(values[0])
        list2.append(values[1])

#tuplesList = zip(sorted(list1), sorted(list2))
#
#result = 0
#for x in tuplesList:
#    result += abs(int(x[0])-int(x[1]))

#print(result)
result2 = 0
for x in list1:
    repetitions = list2.count(x)
    result2 += int(x) * repetitions
print(result2)